src/Raven.Server/Program.cs

```diff
@@ -277,7 +277,8 @@ private static bool RunInteractive(RavenServer server)
             //stop dumping logs
             LoggingSource.Instance.DisableConsoleLogging();
 
-            return new RavenCli().Start(server, Console.Out, Console.In, true);
+            var consoleColoring = !CommandLineSwitches.NonConsole;
+            return new RavenCli().Start(server, Console.Out, Console.In, consoleColoring);
```

Comment:
What is the issue with coloring?

Reply:
When we use the embedded we redirect the stdin output.
When coloring is true we do some checks when we try to write something and then we get an exception.